src/tools/material.js

```diff
@@ -32,7 +34,26 @@ function scaleLatticeToMakeNonPeriodic(material) {
     });
 }
 
+/**
+ * Updates the basis of a material by translating the coordinates
+ * so that the center of the material and lattice are aligned.
+ * @param material {Material}
+ * */
+function getBasisConfigTranslatedToCenter(material) {
+    const lattice = new Lattice(material.lattice);
+    const basis = new Basis({
+        ...material.Basis.toJSON()
+    });
+    basis.toCartesian();
+    const centerOfCoordinates = basis.centerOfCoordinatesPoint;
```

Comment:
Seems like too much code for what it needs to be

Why not just work on the `material.Basis` directly? There's also `material.toCartesian` function used above in the file...